packages/openssl.rb

```diff
@@ -65,10 +65,32 @@ def self.check
     return if ARCH == 'i686'
 
     # Don't run tests if we are just rebuilding the same version of openssl.
-    system 'make test' unless `openssl version | awk '{print $2}'`.chomp == '1.1.1s'
+    system 'make test' unless `openssl version | awk '{print $2}'`.chomp == version.to_s
   end
 
   def self.install
     system "make DESTDIR=#{CREW_DEST_DIR} install_sw install_ssldirs"
+    # Extract OpenSSL 1.1.1t libraries for backwards compatibility purposes.
+    # Builds and rebuilds of packages against OpenSSL should automatically
+    # build against OpenSSL 3.x and not against OpenSSL 1.1.1x.
+    File.write 'openssl111_files', <<~EOF
+      usr/local/#{ARCH_LIB}/libcrypto.so.1.1
+      usr/local/#{ARCH_LIB}/libssl.so.1.1
```

Comment:
What happens if `CREW_PREFIX` is something other than `/usr/local`?

Reply:
I think we have torn out the plumbing to enable that in any case... but open to code to just remove the first slash from CREW_PREFIX...

Reply:
This is definitely an example of tearing out the plumbing. ;)

Reply:
`CREW_PREFIX.chr` might fix this...